docs/topic/credits.md

```diff
@@ -0,0 +1,48 @@
+# Cloud Credits
+
+On each cloud provider, it is common to provide cloud credits that cover the infrastructure costs instead of paying with cash.
+This page has information about how this works for each major cloud provider we use.
+
+## Amazon Web Services
+
+On Amazon Web Services, credits come via **Promotion Codes**.
+These are unique strings that are attached to a given credit offer.
+We can apply these codes to any Account in AWS.
+To do so, follow these steps:
+
+1. Log in to the relevant account via [our AWS SSO](cloud-access:aws-sso).
+2. Confirm that you are in the right account! Credits will *only be available for this account*, and unavailable in any other accounts in the 2i2c AWS organization.
+3. Go to {guilabel}`AWS Billing` -> {guilabel}`Credits`
+4. Click {guilabel}`Redeem Credit` in the top-right.
+5. Enter the Promotion Code and click {guilabel}`Redeem Credit` in the bottom-right.
+
+### Expiration dates
```

Comment:
```suggestion
5. Enter the Promotion Code and click {guilabel}`Redeem Credit` in the bottom-right.

:::{warning}
Credits will be applied to the **next billing period** (generally the next calendar month).
If you apply credits in the middle of the month, they will not cover any of that month's usage.
:::

### Expiration dates
```

Reply:
how's that @yuvipanda ?

Reply:
@choldgraf sounds good! Can you also add the aws docs link the support person sent us?

Reply:
how's that?

Reply:
@choldgraf looks great! Merge whenever you feel like